src/main/java/no/unit/nva/testutils/MockContext.java

```diff
@@ -0,0 +1,80 @@
+package no.unit.nva.testutils;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.amazonaws.services.lambda.runtime.ClientContext;
+import com.amazonaws.services.lambda.runtime.CognitoIdentity;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import java.util.UUID;
+
+public class MockContext implements Context {
```

Comment:
Why not simply making `FakeContext` instead of having to write mocks? Then we inject the `FakeContext` in tests?

Reply:
Is this version closer to what you are thinking?

Reply:
So when I'am thinking about mocks, I'am thinking something along the lines like this:

```
var contextMock = mock(Context.class) // interface
when(contextMock.getRequestId).thenReturn(id)
```

So you write expectations on a interface, usually something you cannot connect directly to use. So you write an expectation to avoid testing that component, but you have certain expectations.

So for me, this class is a `Fake`, which is not a real implementation, but good enough doing the job in tests. 

Example>

Interface DatabaseReader. 
Implementation DynamodbReader of DatabaseReader , which say we couldn't use in the tests because we werent able to spin up a local dynamodb instance. 
Instead of writing test expectations, we would provide a `fake`, this case `FakeInmemoryReader` of DatabaseReader, which would have had a simple implementation and just keeping data in `Map<String, Object>` or something. 

For me, the `MockContext` here is a `FakeContext`.

Reply:
I did not realize that the naming was the problem. Renamed to `FakeContext`, although one can in practice override the methods and have the functionality of the mock.